README.md

```diff
@@ -0,0 +1,153 @@
+
+#### Examples
+
+To align view `a` to be 10 points to the left of view `b`:
+
+```swift
+  a.rightCenter --> 10.horizontalOffset + b.leftCenter
+```
+
+To align view `a` to be 10 points to the right of view `b`:
+
+```swift
+  b.rightCenter + 10.horizontalOffset <-- a.leftCenter
+```
+
+To align view `a` to be 10 points below its superview’s top center:
+
+```swift
+  a.topCenter --> 10.verticalOffset + .topCenter
+```
+
+For more examples, check out the [ViewPositionVisualization](RelativityVisualization.playground/Pages/ViewPositionVisualization.xcplaygroundpage/Contents.swift) playground page.
+
+#### Anchoring UILabels
+
+Relativity makes it easy to position your `UILabel`s to your designer’s spec. Design teams (and design products like [Sketch](https://www.sketchapp.com) and [Zeplin](https://zeplin.io)) measure the vertical distance to a label using the font’s [cap height](https://en.wikipedia.org/wiki/Cap_height) and [baseline](https://en.wikipedia.org/wiki/Baseline_(typography)). Relativity’s `align` methods measure `UILabel` distances the same way. So if your spec says that label `b` should be eight vertical points below label `a`, all you need is:
+
+```swift
+  a.bottomCenter --> 8.verticalOffset + b.topCenter
+```
+
+For a visual example, check out the [FontMetricsVisualization](RelativityVisualization.playground/Pages/FontMetricsVisualization.xcplaygroundpage/Contents.swift) playground page.
+
+### Flexible distribution of subviews
+
+Since view controllers need to be laid out flexibly over various iOS device sizes, Relativity has the ability to easily distribute subviews with flexible positioning along an axis.
+
+Subview distribution can be controlled by positioning fixed and flexible spacers in between views. Fixed spaces represent points on screen. Fixed spaces are created by inserting `CGFloatConvertible` (`Int`, `Float`, `CGFloat`, or `Double`) types into the distribution expression, or by initializing a `.fixed(CGFloatConvertible)` enum case directly. Flexible spacers represent proportions of the remaining space in the superview after the subviews and fixed spacers have been accounted for. You can create flexible spacers by surrounding an `Int` with a spring `~` operator, or by initializing the `.flexible(Int)` enum case directly. A `~2~` represents twice the space that `~1~` does. Views, fixed spacers, and flexible spacers are bound together by a bidirectional anchor `<>` operator.
+
+#### Examples
+
+To equally distribute subviews `a`, `b`, and `c` at equal distances along a horizontal axis:
+
+```swift
+  superview.distributeSubviewsHorizontally() {
+    a <> ~1~ <> b <> ~1~ <> c
```

Comment:
Probably not worth it, but as a thought exercise, could you get compile time constraints by defining the operators something like below.

Should work: `1 - a ~ 2 ~ b`
Should not work: `1 ~ 8 - a ~ 2` 
Should not work: `a - 8 ~ b`

```
infix func - (left: CGFloatConvertible, right: UIView) -> ViewPosition   // Handles "1 - a"
infix func ~ (left: ViewPosition, right: CGFloatConvertible) -> IncompleteFlexibleViewOperation // Handles "(1 - a) ~ 2"
infix func ~ (right: IncompleteFlexibleViewOperation, right: UView) -> ViewPosition // Handles "((1 - a) ~ 2) ~ b"
```

Note that the use of `IncompleteFlexibleViewOperation` means that no operator overload exists that would allow the other combinations to work.